packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -47,10 +55,86 @@ class CloudFormation {
 
       this.cfn = new aws.CloudFormation({ ...cred, ...options, ...userAgentOption });
       this.context = context;
+      if (Object.keys(eventMap).length) {
+        this.eventMap = eventMap;
+        this.progressBar = this.initializeProgressBars();
+      }
       return this;
     })();
   }
 
+  // Custom Item formatter for progresbar
+  createItemFormatter(payload) {
```

Comment:
since this file is already enormous, can we break this into a new file such as `aws-cfn-progress-formatter`?